### PR TITLE
SWING-150 removed django_cms, drifter and vagrant

### PR DESCRIPTION
**Pull request by T4m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,6 @@ To create a new project using this template, do the following:
...
 
-If you chose to use [Drifter](https://github.com/liip/drifter), run the
-following commands to get you project up and running:
-
-    vagrant up
-    vagrant ssh
-    ./manage.py createsuperuser
-    ./manage.py runserver
-
 Go to http://my-project.lo/ and start hacking!
```
Comment (felicemcc) [inline README.md]: http://my-project.dockert.test/

**Comment by felicemcc** [comment]
Hello, I found as well:

**File: django-template/{{cookiecutter.project_slug}}/.gitignore**

- Still references to drifter and vagrant


**File: django-template/hooks/post_gen_project.py**

- Please remove install_drifter() function

- Look for these lines and remove them:
```
set_parameter(path, 'box_name', 'drifter/stretch64-base')
set_parameter(path, 'box_url', 'https://vagrantbox-public.liip.ch/drifter-stretch64-base.json')

```

**File: django-template/{{cookiecutter.project_slug}}/setup.cfg**

- References to drifter

File: README.md (hunk 1 above)
Comment (Nadeempoco) [reply README.md]: this is the link we should use?

File: README.md [hunk 2]
```diff
@@ -6,15 +6,7 @@ To create a new project using this template, do the following:
...
-If you chose to use [Drifter](https://github.com/liip/drifter), run the
-following commands to get you project up and running:
-
-    vagrant up
-    vagrant ssh
-    ./manage.py createsuperuser
-    ./manage.py runserver
-
-Go to http://my-project.lo/ and start hacking!
+Go to http://my-project.dockert.test/ and start hacking!
```
Comment (danickfort) [inline README.md]:
```suggestion
Go to http://my-project.docker.test/ and start hacking!
```

File: {{cookiecutter.project_slug}}/setup.cfg [hunk 3]
```diff
@@ -16,7 +16,7 @@ known_django = django
 default_section = THIRDPARTY
```
Comment (danickfort) [inline {{cookiecutter.project_slug}}/setup.cfg:16]: On line 6 of this file : Remove `{% if cookiecutter.virtualization_tool == 'drifter' %}virtualization/, {% endif %}`

**Review by danickfort — CHANGES_REQUESTED** [review]
@T4m @Nadeempoco Some small changes

**Comment by danickfort** [comment]
looks good to me, I would merge the PR if I had permissions 😄  👍

**Pull request merged**